### ISSUE TITLE
Remove invalid candidates

### ISSUE THREAD

**Issue by mxmehl** [tonarino/innernet#198]
We recently removed the IPv4 of two dual-stack servers (let's call them A and B) connected to innernet. Most peers continue to connect to the server just fine (via IPv6), but one (C in this example) still tries to connect to both of them via IPv4.

The innernet server itself is IPv6 only. C is dual-stack.

We tried numerous things: restarting the affected servers and also updating the `candidates` column in the network's database on the innernet server via `UPDATE peers SET candidates='[]' where name='A';`. All efforts only work for a few seconds, then it revert back to the unavailable IPv4 addresses.

Is there some way to remove a candidate altogether from the server and all its peers, especially when it's unavailable?

**Comment by mcginty** [comment]
A peer's candidates are overwritten with each run of `innernet up` from the peer (so every 60 seconds if using the provided systemd unit).

If you run `innernet` with the `-v` flag:
```
sudo innernet -v up [mynetwork]
```

you'll see more detailed information about the endpoints that the clients are trying when doing NAT traversal. Perhaps that output will help you debug. Also, double-check that you didn't use `innernet override-endpoint` to force an endpoint for peers A and B.

**Comment by mxmehl** [comment]
Thanks for the command, I forgot about using `-v` (or `-vv`) but it helped me to better understand the candidate reporting method.

I fixed the problem, and it was a stupid combination. Pehraps this helps people who run into similar issues:

1. One host still showed the IPv4 (e.g. in `ip a`) although it was removed from the config. When I fixed that, innernet indeed removed the candidate immediately.
2. The other host had a mistake in the firewall configuration. It was not able to make outgoing requests via HTTP to the innernet server, but pings and plain wireguard worked. When I fixed that, the problem also was gone.
